<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NorduGrid ARC Job manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <zorder>logText</zorder>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTextEdit" name="logText">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenJobList"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>Jobs</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionKillSelected"/>
    <addaction name="actionCleanSelected"/>
    <addaction name="actionResubmitSelected"/>
    <addaction name="actionDownloadSelected"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="actionClearSelection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuJobs"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenJobList"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionKillSelected"/>
   <addaction name="actionCleanSelected"/>
   <addaction name="actionResubmitSelected"/>
   <addaction name="actionDownloadSelected"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="arcstat-ui.qrc">
     <normaloff>:/arcstat-ui/images/reload.png</normaloff>:/arcstat-ui/images/reload.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionKillSelected">
   <property name="icon">
    <iconset resource="arcstat-ui.qrc">
     <normaloff>:/arcstat-ui/images/gtk-cancel.png</normaloff>:/arcstat-ui/images/gtk-cancel.png</iconset>
   </property>
   <property name="text">
    <string>Kill</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCleanSelected">
   <property name="icon">
    <iconset resource="arcstat-ui.qrc">
     <normaloff>:/arcstat-ui/images/gtk-clear.png</normaloff>:/arcstat-ui/images/gtk-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionResubmitSelected">
   <property name="icon">
    <iconset resource="arcstat-ui.qrc">
     <normaloff>:/arcstat-ui/images/edit-redo.png</normaloff>:/arcstat-ui/images/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Resubmit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDownloadSelected">
   <property name="icon">
    <iconset resource="arcstat-ui.qrc">
     <normaloff>:/arcstat-ui/images/document-save.png</normaloff>:/arcstat-ui/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpenJobList">
   <property name="icon">
    <iconset resource="arcstat-ui.qrc">
     <normaloff>:/arcstat-ui/images/document-open.png</normaloff>:/arcstat-ui/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open job list...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>SelectAll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionClearSelection">
   <property name="text">
    <string>Clear selection</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="arcstat-ui.qrc"/>
 </resources>
 <connections/>
</ui>
